api/v1alpha1/dragonfly_types.go

```diff
@@ -114,6 +114,11 @@ type DragonflySpec struct {
 	// +kubebuilder:validation:Optional
 	Snapshot *Snapshot `json:"snapshot,omitempty"`
 
+	// (Optional) Whether running on Openshift or not
+	// +optional
+	// +kubebuilder:validation:Optional
+	Openshift bool `json:"openshift,omitempty"`
```

Comment:
Maybe can you rename this to `skipFSGroup`? which makes better sense than Openshift? LGTM otherwise! 👍🏼

Reply:
Updated: 
https://github.com/ArthurVardevanyan/HomeLab/commit/ce549e5710b3ad79e484a0e583b812634e3378df